internal/flink/command_statement_list.go

```diff
@@ -1,12 +1,27 @@
 package flink
 
 import (
+	"fmt"
+	"slices"
+	"strings"
+
 	"github.com/spf13/cobra"
 
 	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
 	"github.com/confluentinc/cli/v3/pkg/output"
 )
 
+var allowedStatuses = []string{
```

Comment:
This should come from an enum somewhere

Reply:
I wish our SDKs could generate lists of enums... since it's possible that this list gets out of sync with the backend, it seems like a good idea to only use this list for autocomplete, and not for erroring on invalid flag values.

Reply:
> it seems like a good idea to only use this list for autocomplete, and not for erroring on invalid flag values.

The issue I foresee is a user inputting an invalid status, and just getting a None found.

How about logging a warning if the provided status is not recognised (instead of erroring)?

Reply:
:+1: